### ISSUE TITLE
NamedExec unable to bulk insert with functions

### ISSUE THREAD

**Issue by Mungrel** [jmoiron/sqlx#713]
Example query:

```sql
INSERT INTO some_table (
 name,
 value
) VALUES (
 :name,
 handle_value(:value)
)
```

It looks to be due to [the regex](https://github.com/jmoiron/sqlx/blob/master/named.go#L227) matching the opening bracket of the `handle_value` function.

**Comment by Mungrel** [comment]
I don't think there's any way to solve this effectively with Regex, and instead the parentheses should be parsed

**Comment by ddouglas** [comment]
Yeah, we have this same issue when using `CURDATE()` & `NOW()`

**Comment by yyh-gl** [comment]
`UUID_TO_BIN()` too 📝

**Comment by jmoiron** [comment]
fixed in v1.3.2
